web/src/main/scala/org/hyperscala/io/HTMLToScala.scala

```diff
@@ -32,17 +33,19 @@ object HTMLToScala {
     parent.read(body)                                  // Read the new data back in
   }
 
-  def toScala(page: Webpage, packageName: String, className: String) = {
+  def toScala(page: tag.HTML, packageName: Option[String], className: String): String = {
```

Comment:
If you're going to change the type here you should also change the variable name `page` to something like `html`.